RetsTalk/RetsTalk/RetrospectCalendar/Controller/RetrospectCalendarTableViewController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  RetrospectCalendarTableViewController.swift
+//  RetsTalk
+//
+//  Created by KimMinSeok on 12/1/24.
+//
+
+import SwiftUI
+import UIKit
+
+final class RetrospectCalendarTableViewController: BaseViewController {
+    private typealias DataSource = UITableViewDiffableDataSource<Section, Retrospect>
+    private typealias Snapshot = NSDiffableDataSourceSnapshot<Section, Retrospect>
+    
+    private var dataSource: DataSource?
+    private var snapshot: Snapshot?
+    
+    private var retrospects: [Retrospect]
+    
+    // MARK: View
+    
+    private let retrospectCalendarTableView = RetrospectCalendarTableView()
+    
+    // MARK: Initalization
+    
+    init(retrospects: [Retrospect]) {
+        self.retrospects = retrospects
+        
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: RetsTalk Lifecycle
+    
+    override func loadView() {
+        view = retrospectCalendarTableView
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    
+        updateTableView()
+    }
+    
+    override func setupDataSource() {
+        super.setupDataSource()
+        
+        dataSource = DataSource(
+            tableView: retrospectCalendarTableView.retrospectListTableView
+        ) { tableView, indexPath, retrospect in
+            let cell = tableView.dequeueReusableCell(
+                withIdentifier: Constants.Texts.retrospectCellIdentifier,
+                for: indexPath
+            )
+            self.configureCell(cell: cell, retrospect: retrospect)
+            return cell
+        }
+    }
+    
+    private func configureCell(cell: UITableViewCell, retrospect: Retrospect) {
+        cell.selectionStyle = .none
+        cell.backgroundColor = .clear
+        cell.contentConfiguration = UIHostingConfiguration {
+            RetrospectCell(
+                summary: retrospect.summary ?? Texts.defaultSummaryText,
+                createdAt: retrospect.createdAt,
+                isPinned: retrospect.isPinned
+            )
+        }
```

Comment:
```suggestion
        cell.contentConfiguration = UIHostingConfiguration {
            RetrospectCell(
                summary: retrospect.summary ?? Texts.defaultSummaryText,
                createdAt: retrospect.createdAt,
                isPinned: retrospect.isPinned
            )
        }
        .margin(.vertical, 8.0)
```

와 같이 마진을 추가하면 셀 간 간격이 관리됩니다.